crates/red_knot/tests/cli.rs

```diff
@@ -244,6 +350,25 @@ fn unknown_rules() -> anyhow::Result<()> {
     Ok(())
 }
 
+/// Red Knot warns about unknown rules specified in a CLI argument
+#[test]
+fn cli_unknown_rules() -> anyhow::Result<()> {
+    let case = TestCase::with_file("test.py", "print(10)")?;
+
+    case.insta_settings().bind(|| {
+        assert_cmd_snapshot!(case.command().arg("--ignore").arg("division-by-zer"), @r"
+        success: false
+        exit_code: 1
```

Comment:
Is it intended that we want to fail with exit code 1 in case of warnings?

Reply:
No, there's another issue that looks into changing this behavior.